docs/en/user_guides/4_pretrain_custom_dataset.md

```diff
@@ -0,0 +1,370 @@
+# Tutorial 4: Pretrain with Custom Dataset
+
+- [Tutorial 4: Pretrain with Custom Dataset](#tutorial-4-pretrain-with-custom-dataset)
+  - [Train MAE on Custom Dataset](#train-mae-on-custom-dataset)
+    - [Get the path of custom dataset](#get-the-path-of-custom-dataset)
+    - [Choose one config as template](#choose-one-config-as-template)
+    - [Edit the dataset related config](#edit-the-dataset-related-config)
+  - [Train MAE on COCO Dataset](#train-mae-on-coco-dataset)
+  - [Train SimCLR on Custom Dataset](#train-simclr-on-custom-dataset)
+  - [Load Pre-trained Model to Speedup Convergence](#load-pre-trained-model-to-speedup-convergence)
+
+In this tutorial, we provide some tips on how to conduct self-supervised learning on your own dataset(without the need of label).
+
+## Train MAE on Custom Dataset
+
+In MMSelfSup, We support the `CustomDataset` from MMClassification,  which is able to read the images within the specified folder directly. You only need to prepare the path information of the custom dataset and edit the config.
```

Comment:
Maybe we can add something like: similar to the ``ImageFolder`` in ``torchvision``.